### ISSUE TITLE
Main window doesn't get focus and menus don't work

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#3711]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1065 (but think I've seen this with earlier 1.2 builds)
    OS Version      : macOS 10.14 (but think I saw this with 10.13 on same machine)
    R Version       : 3.5.1

### Steps to reproduce the problem
* Start RStudio, don't click on the main window when it appears.
* Try to use any menu items, such as Help / About

### Describe the problem in detail
* Menus don't do anything
* Console doesn't have focus until you click on it
* Once you click on the main window, the menus start working

### Describe the behavior you expected
* Console has focus on startup, menus work

### Other details
I have only been able to repro on my Macbook Pro, 15", mid-2014, but it happens 100% of the time. Same version of RStudio and R don't exhibit this problem on my iMac.

Does not happen with 1.1.456.

**Comment by kevinushey** [comment]
For posterity, how are you launching RStudio? (I can't reproduce on my Mojave machine)

**Comment by gtritchie** [comment]
After launching and making one try at Help / About RStudio, checked the JavaScript console and saw this:
![2018-10-19_21-49-20](https://user-images.githubusercontent.com/10569626/47251384-f587ef00-d3e8-11e8-92eb-5a23f9596753.png)

**Comment by gtritchie** [comment]
I was able to repro with a development build, using ant desktop. However, I then resized the main RStudio window (going from taking up most of the screen to about half the screen), then restarted and the problem stopped happening. Then I tried again with 1.2.1065, and the problem was no longer happening. I have not been able to repro since.

**Comment by gtritchie** [comment]
I can consistently repro.

Close all RStudio instances, then run this script:
```
#!/usr/bin/env bash

defaults delete com.rstudio.desktop
defaults delete com.rstudio.desktopPro
defaults delete org.rstudio.RStudio
rm -rf ~/rstudio-desktop
```
Now launch RStudio by double-clicking app or via Spotlight.

The console cursor is greyed out (doesn't have focus) and the menus don't do anything. Clicking on the main window fixes it. In this case, close and restarting the application fixes the problem on subsequent launches; I've had cases where it continued to exhibit the problem on every launch until I resized the window before exiting.

Attached video showing IDE after first launch with the problems, then clicking in console and trying menus again.

![2018-11-04_13-13-24](https://user-images.githubusercontent.com/10569626/47969955-df5a6f80-e033-11e8-9005-612551f60089.gif)

**Comment by kevinushey** [comment]
Should be fixed with https://github.com/rstudio/rstudio/commit/ed2d415589859941295fa5e146fe4a2599612e69.

**Comment by gtritchie** [comment]
I still repro the problem (running a dev build with this change; I'll try again once the daily with this is available).

No longer seeing the error in the JavaScript console, but the behavior still happens (no focus in console, menus are no-ops).

**Comment by gtritchie** [comment]
I can no longer repro, closing it.

**Comment by gtritchie** [comment]
This is still happening. Currently getting this 100% of the time when I start version 1.2.1578 on macOS 10.14.6.

If I turn on VoiceOver (Cmd+F5 on normal keyboard, Cmd+Triple-tap Touch ID button on touchbar keyboard), then start RStudio, focus goes to the main window frame, but not into it. Hitting Tab key will then get things into an active state, keyboard-wise (ignore the fact VoiceOver doesn't say what focus is on, that's a whole other nest of vipers).

**Comment by gtritchie** [comment]
This appears to be related to Qt 5.12.0 in some way:

- Build 1.2.1206 (last build with 5.11.1) did not exhibit this problem.
- Build 1.2.1207+ (first build with 5.12.0) does have this problem.
